include/bitcoin/system/impl/data/collection.ipp

```diff
@@ -254,7 +254,9 @@ Collection difference(const Collection& minuend,
     const Collection& subtrahend) noexcept
 {
     Collection copy;
-    copy.reserve(std::size(minuend));
+
+    // Replaced std::size due absence, C++17 feature
+    copy.reserve(minuend.size());
```

Comment:
This should only be called with std::vector, which should be supported by std::size?

Reply:
std::size is documented as a C++17 addition. It is not available in C++11 for GCC.

Reply:
Ah, forgot that. I must have stripped the others out based on CI a while back.